crud.py

```diff
@@ -0,0 +1,99 @@
+from sqlalchemy.orm import Session
+
+from db import models
+from schemas import AuthorCreate, BookCreate
+
+
+def create_author(db: Session, author: AuthorCreate) -> models.Author:
+    db_author = models.Author(**author.model_dump())
+    db.add(db_author)
+    db.commit()
+
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_author(db: Session, author_id: int) -> models.Author:
+    return (
+        db.query(models.Author).filter(models.Author.id == author_id).first()
+    )
+
+
+def get_author_list(
+    db: Session, skip: int = 0, limit: int = 10
+) -> list[models.Author]:
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def update_author(
+    db: Session, author_id: int, author: AuthorCreate
+) -> models.Author:
+    db_author = (
+        db.query(models.Author).filter(models.Author.id == author_id).first()
+    )
+    if db_author:
+        for key, value in author.dict().items():
```

Comment:
The method `dict()` is used here, but it might not exist in the `AuthorCreate` schema. Ensure that `AuthorCreate` has this method or replace it with the appropriate method to convert the schema to a dictionary.